contrib/rpcuser/rpcuser.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
```

Comment:
nit: If this script is supposed to be used by end users, it should be in `share/` (I think) and be installed with `make install`. Contrib is usually for obscure things only useful for developers.